src/driver/amdxdna/aie2_ctx.c

```diff
@@ -601,7 +614,6 @@ static int aie2_ctx_syncobj_create(struct amdxdna_hwctx *hwctx)
 		return ret;
 	}
 	ret = drm_syncobj_get_handle(filp, syncobj, &hdl);
-	drm_syncobj_put(syncobj);
```

Comment:
i thought you need to move this line down into the error handling code path, no?

Reply:
Oops. I forgot to move.